frontend/projects/setup-wizard/src/app/pages/recover/recover.page.html

```diff
@@ -25,13 +25,13 @@
             <!-- loaded -->
             <ion-item-group *ngIf="!loading" class="ion-text-center">
               <!-- cifs -->
-              <h2 class="target-label">Shared Network Folder</h2>
+              <h2 class="target-label">LAN Shared Folder</h2>
               <p class="ion-padding-bottom">
-                Using a shared folder is the recommended way to recover from
+                Using a LAN shared folder is the recommended way to recover from
```

Comment:
```suggestion
                Using a LAN Shared Folder is the recommended way to recover from
```

To instill that this is "formal terminology" we will be using throughout the UX.